content/en/docs/tasks/security/https-overlay/index.md

```diff
@@ -161,12 +161,10 @@ $ kubectl get pod -n istio-system
 No resources found.
 {{< /text >}}
 
-Follow the [installation instructions](/docs/setup/install/kubernetes/#installation-steps) install CRDs.
-
 Install Istio with the **strict mutual TLS mode** enabled:
 
 {{< text bash >}}
-$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-demo.yaml --set global.controlPlaneSecurityEnabled=true --set global.mtls.enabled=true | kubectl delete -f -
+$ istioctl manifest --set profile=demo --set values.global.controlPlaneSecurityEnabled=true --set values.global.mtls.enabled=true | kubectl delete -f -
```

Comment:
This is strange - we're piping to delete?

Reply:
@ostromart this was `manifest generate` iirc - but generate is not present.  I think there is a prune feature, but I have not made much use of the CLI mode.

Reply:
It's confusing because this is an install step and we were deleting here, not sure why.

Reply:
Yeah, it makes no sense. I change it to `istioctl manifest apply` and removed the pipe to `kubectl delete`.

Reply:
right - I didn't actually read the rest of the context.  looks like an existing defect in the docs.